docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -62,3 +63,70 @@ Log in the back office as an Administrator and go to the Users page. Select the
 
 Then log in again into the back office with your new user credentials.
 You have now the possibility to create new Rides and Points of Interest only.
+
+
+
+##  Customize the use forms
+
+We want to have a consistent design over our web site. When using the register form, it was a raw HTML form without any styling on it. We want to welcome our new users with a pretty register page.
+
+### Use a views.yml file
+
+In order to be clearer, we will extract the YAML directives regarding the presentation and put them into their own separate file. Create the views.yml file and cut-paste into it the parts of ezplatform.yml that are about the templating.
+
+```yaml
+ezpublish:
+    system:
+        default:
+            pagelayout: pagelayout.html.twig
+            content_view:
+                    full:
+                        full_ride:
+                            template: "full/ride.html.twig"
+                            controller: "AppBundle:Ride:viewRideWithPOI"
+                            match:
+                                Identifier\ContentType: "ride"
+                        root_folder:
+                            template: "full/root_folder.html.twig"
+                            match:
+                                Id\Location: 2
+                    line:
+                        line_point_of_interest:
+                            template: 'line/point_of_interest.html.twig'
+                            match:
+                                Identifier\ContentType: ['point_of_interest']
+                        line_ride:
+                            template: "line/ride.html.twig"
+                            match:
+                                Identifier\ContentType: "ride"
+```
+
+And at the beginnning of the `ezplatform.yml` file, add the following lines to import the content of views.yml:
+
+```yaml
+imports:
+    - { resource: views.yml }
+```
+
+!!! note
+
+```
+You can clear the cache with the command we have seen before: `php app/console cache:clear` to check that nothing has changed.
```

Comment:
`nothing has changed` sound a bit as if the change we've done did not work... Maybe replace with something like `to check that the configuration still works like before`?